.travis.yml

```diff
@@ -1,6 +1,7 @@
+before_install: gem install bundler -v '1.15.0'
```

Comment:
This is because bundler v1.15.1 has a bug. https://github.com/bundler/bundler/issues/5373#issuecomment-313086161
Wait till it is released?